kubectl-openebs/cli/command/get/volume_list.go

```diff
@@ -32,39 +31,28 @@ This command displays status of available zfs Volumes.
 If no volume ID is given, a list of all known volumes will be displayed.
 
 Usage: kubectl openebs cStor volume list [options]
-	`
-	namespace string
+`
 )
 
-const (
-	volumeStatusOK = "Running"
-)
-
-// NewCmdVolumesList displays status of OpenEBS Volume(s)
-func NewCmdVolumesList() *cobra.Command {
+// NewCmdGetVolume displays status of OpenEBS Volume(s)
+func NewCmdGetVolume() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "list",
-		Short: "Displays status information about Volume(s)",
-		Long:  volumesListCommandHelpText,
+		Use:     "volume",
+		Aliases: []string{"vol", "v", "volumes"},
+		Short:   "Displays status information about Volume(s)",
+		Long:    volumesListCommandHelpText,
 		Run: func(cmd *cobra.Command, args []string) {
 			util.CheckErr(RunVolumesList(cmd), util.Fatal)
 		},
 	}
-
-	cmd.Flags().StringVarP(&namespace, "namespace", "n", "openebs",
-		"namespace name, required if volume is not in the `default` namespace")
-
-	flag.CommandLine.Parse([]string{})
-
 	return cmd
 }
 
-//RunVolumesList fetchs & lists the volumes
+// RunVolumesList lists the volumes
 func RunVolumesList(cmd *cobra.Command) error {
-
-	client, err := client.NewK8sClient(namespace)
+	// TODO: Why is this working?
+	client, err := client.NewK8sClient("")
 	util.CheckErr(err, util.Fatal)
```

Comment:
I tried this by installing openebs on the test namespace and this still worked, I believe "" used to be the k8s constant for default or all-namespaces(my memory is a lot hazy here).

Reply:
if it working with `namespace` then I think it's good to keep that.

I am assuming `namespace` is an arg here provided by the cli user?

Reply:
We can have a universal function to return the `client` using the `namepsace` as I see 
`client, err := client.NewK8sClient()` is repeated a couple of times. wdyt?

Reply:
This is a great idea, this will make writing tests easier if it's initialized in the `openebs` sub-command level and passed around to the sub-sub-commands which'd need them.

Reply:
> if it working with namespace then I think it's good to keep that.

Yeah, but then if it was working with the namespace, it should likely break without it. I'll try to understand how this is still working.

> I am assuming namespace is an arg here provided by the cli user?

Yes, the flags nee to be defined at a command level, I saw they were defined multiple times(locally) so I created a PersistentFlag and then removed it because it's likely already coming from `kubectl-openebs/cli/command/commands.go`  line#40 in my PR.

```go
	kubernetesConfigFlags := genericclioptions.NewConfigFlags(true)
	kubernetesConfigFlags.AddFlags(cmd.PersistentFlags())
```

Reply:
> We can have a universal function to return the client using the namespace as I see

I'll do that in upcoming PRs, this could be a big refactor, because I also need to support other quirks correctly(e.g. the end user passes the `-u` flag or `--kubeconfig` flag, etc. IMO, this should be initialized at the `openebs` command level and maybe passed around to other sub-command objects.